scripts/build_gh_pages

```diff
@@ -29,46 +30,37 @@ Auto-generated from commit ${BRANCH_SHA1} ("${BRANCH_SUBJECT}").
 END
 )
 
-tgz_gen() {
+doc_gen() {
   local srcdir=$1
   local docdir=$2
-  local tarfile="/tmp/oak-doc-${docdir}-${BRANCH_SHA1}.tgz"
-  # Build Cargo docs and save them off outside the repo
   (
     cd "${srcdir}" || exit
-    rm -rf target/doc
     cargo doc --no-deps
     cargo deadlinks
-    cd target/doc || exit
-    tar czf "${tarfile}" ./*
+    cp --recursive ./target/doc "${TARGET_DIR}/${docdir}"
   )
-  echo "${tarfile}"
 }
 
-tarfiles=()
 for ii in "${!SRC_SUBDIRS[@]}"; do
-   tarfiles+=( "$(tgz_gen "${SRC_SUBDIRS[$ii]}" "${DOC_SUBDIRS[$ii]}")" )
+  doc_gen "${SRC_SUBDIRS[$ii]}" "${DOC_SUBDIRS[$ii]}"
 done
 
-# Shift to gh-pages branch and replace contents
-git checkout gh-pages
-
-doc_update() {
-  local docdir=$1
-  local tgzfile=$2
-  rm -rf "${docdir}"
-  mkdir "${docdir}"
-  (
-    cd "${docdir}" || exit
-    tar xzf "${tgzfile}"
-  )
-}
-
-for ii in "${!DOC_SUBDIRS[@]}"; do
-  doc_update "${DOC_SUBDIRS[$ii]}" "${tarfiles[$ii]}"
-done
+cat <<-END > "${TARGET_DIR}/index.html"
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Refresh" content="0; url=./sdk/oak/index.html" />
+  </head>
+  <body>
+    <p><a href="./sdk/oak/index.html">Oak SDK main page</a></p>
+  </body>
+</html>
+END
 
-# Commit any differences
-git add "${DOC_SUBDIRS[@]}"
-git commit --message="${COMMIT_MESSAGE}"
-git checkout "${BRANCH}"
+(
+  cd "${TARGET_DIR}"
+  git add "${DOC_SUBDIRS[@]}"
+  git commit --message="${COMMIT_MESSAGE}"
+  echo "to push changes, run the following command:"
+  echo "git push ${TARGET_DIR}"
```

Comment:
I don't think this will work (I *think* it will push whatever the current local branch is (`master` or `wip` or whatever) to be a new branch in the `/tmp/project-oak-gh-pages-xxxx` codebase).